src/user/auth.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Inject, Injectable, Render } from '@nestjs/common';
+import { Model } from 'mongoose';
+import { sign } from 'jsonwebtoken';
+import * as bcrypt from 'bcryptjs';
+import { UserSchema } from '../schemas/user.schema';
+import { userLoginDTO } from './model/user.model';
+
+@Injectable()
+export class authService {
+  constructor(
+    @Inject('USER_MODEL')
+    private userModel: Model<typeof UserSchema>,
+  ) {}
+
+  public async loginService(userLoginDTO) {
```

Comment:
This function can be named `login` or `userLogin`
```suggestion
  public async userLogin(userLoginDTO: userLoginDTO) {
```

`Service` is a keyword generally reserved for Service classes